ui/nuxeo-data-table/data-table-cell.js

```diff
@@ -79,7 +79,11 @@ import './data-table-templatizer-behavior.js';
 
     ready() {
       super.ready();
-      this.setAttribute('role', 'cell');
+      if (this.header !== undefined) {
```

Comment:
```suggestion
      if (!this.header) {
```

Reply:
(!this.header) doesn't work hence used (this.header)

Reply:
Did you understand why it doesn't work?

Reply:
Yes Sam, if we use !this.header header elements will not be included